src/brucke_config.erl

```diff
@@ -303,29 +303,35 @@ validate_client_config(ClientId, Config) ->
                   [ClientId, Config]),
   exit(bad_client_config).
 
+%% @private
 do_validate_client_config(ClientId, {ssl, Options}) ->
-  {ssl, validate_ssl_files(ClientId, Options)};
+  {ssl, validate_ssl_option(ClientId, Options)};
 do_validate_client_config(_ClientId, {_, _} = ConfigEntry) ->
   ConfigEntry;
 do_validate_client_config(ClientId, Other) ->
   lager:emergency("Unknown client config entry for client ~p,"
                   "expecting kv-pair\nGot:~p", [ClientId, Other]),
   exit(bad_client_config_entry).
 
--spec validate_ssl_files(client_id(), list()) -> list() | none().
-validate_ssl_files(ClientId, SslOptions) ->
+%% @private
+-spec validate_ssl_option(client_id(), true | list()) ->
+        boolean() | list() | none().
+validate_ssl_option(_ClientId, true) ->
+  true;
+validate_ssl_option(ClientId, SslOptions) ->
   lists:foldl(
     fun(OptName, OptIn) ->
-      validate_ssl_files(ClientId, OptIn, OptName)
+      validate_ssl_option(ClientId, OptIn, OptName)
     end, SslOptions, [ {mandatory, cacertfile}
```

Comment:
not really mandatory actually, e.g. one install a commercial cert on kafka brokers and in this case cacert is not necessary